libkbfs/backpressure_disk_limiter.go

```diff
@@ -402,41 +734,36 @@ func defaultGetFreeBytesAndFiles(path string) (int64, int64, error) {
 	return int64(freeBytes), int64(freeFiles), nil
 }
 
-type bdlSnapshot struct {
-	used  int64
-	free  int64
-	max   int64
-	count int64
+func (bdl *backpressureDiskLimiter) getByteFileSnapshotsForTest() (
+	byteSnapshot, fileSnapshot jtSnapshot) {
+	bdl.lock.RLock()
+	defer bdl.lock.RUnlock()
+	return bdl.journalTracker.getByteFileSnapshotsForTest()
 }
 
-func (bdl *backpressureDiskLimiter) getSnapshotsForTest() (
-	byteSnapshot, fileSnapshot bdlSnapshot) {
+func (bdl *backpressureDiskLimiter) getQuotaSnapshotForTest() jtSnapshot {
```

Comment:
Might make sense to group this function with the above, given how tests use these.